ibmcloud/README.md

```diff
@@ -171,14 +263,18 @@ https://cloud.ibm.com/objectstorage/
 
 First, create a COS service instance if you have not create one. Then, create a COS bucket with the COS instance. The COS service instance and bucket names are necessary to upload a custom VM image.
 
-You can use the Terraform template located at [ibmcloud/terraform/cos](./terraform/cos)to use Terraform to create a COS service instance, COS bucket, and IAM AuthorizationPolicy automatically. These resources are configured to store images. Create a `terraform.tfvars` file in the templates directory that includes these fields:
+You can use the Terraform configuration located at [ibmcloud/terraform/cos](./terraform/cos) to use Terraform to create a COS service instance, COS bucket, and IAM Authorization Policy automatically. These resources are configured to store images. Create a `terraform.tfvars` file in the configurations directory that includes these fields:
 
 ```
 ibmcloud_api_key = "<your API key>"
-cos_bucket_name = "<COS bucket name>"
-cos_service_instance_name = "<COS instance name>"
+cos_bucket_name = "<name of the COS bucket to create>"
+cos_service_instance_name = "<name of the COS instance to create>"
 ```
-> **Note:** The environment variable `cos_service_instance_name` in `variables.tf` has the default value `cos-image-instance` which will be used by Terraform if you do not provide a unique value in `terraform.tfvars`.
+
+> **Notes:**
+> - Variables with the same name as variables in the end to end Terraform configuration are as described in that [configuration's parameters](#parameters).
+> - Additional variables and their defaults are defined in the [variables.tf]()./terraform/cos/variables.tf) file for this Terraform configuration.
```

Comment:
It looks like the classic IDE auto-closing a bracket has happened here on `()./terraform/cos/variables.tf)`

Reply:
I suspect you're exactly right. Good spot there. I removed the extra right bracket.